modules/cstrike/cstrike/CstrikeUtils.cpp

```diff
@@ -47,3 +48,22 @@ bool UTIL_CheckForPublic(const char *publicname)
 
 	return false;
 }
+
+void UTIL_StringToLower(const char *str, char *buffer, size_t maxlength)
```

Comment:
Is this really the first place where "string to lower" function was needed and used? There's probably more places in amxx codebase.